src/index.ts

```diff
@@ -98,9 +109,8 @@ const main = async (): Promise<void> => {
 
   workflowCommand.addCommand(operationCommand);
   cli.addCommand(workflowCommand);
-  cli.addCommand(createLintCommand());
 
-  await cli.exitOverride().parseAsync(process.argv);
+  cli.exitOverride().parse(process.argv);
```

Comment:
Wasn't really sure why but the 9 -> 10 commander update the parse async was capturing the exit code, this should be equivalent though